File: samples/Java/Json Store/Cursor/Select/By Parent.java
Store store = session.getDatabase("playground").getStore("forum");
String s = ""; final StringBuilder b = new StringBuilder();

// Get the first forum document
Document doc = store.openCursor().hierarchical(1).findOne().loadDocument();
String unid = doc.getUnid();

s += ">>> Select all documents by parent UNID, "+unid+"\n";
Cursor c = store.openCursor().range(0,5);

b.setLength(0);
c.parentUnid(unid).find(new CursorHandler() {
  public boolean handle(CursorEntry e) throws JsonException {  
  	b.append("  value:"+e.getValue()+"\n");
    return true;
  }
});  
s += b.toString()+"\n";

s += ">>> Select all documents using parent document\n";
Cursor c2 = store.openCursor().range(0,5);
b.setLength(0);
c2.parent(doc).find(new CursorHandler() {
  public boolean handle(CursorEntry e) throws JsonException {  
  	b.append("  value:"+e.getValue()+"\n");
    return true;
  }
});  
s += b.toString()+"\n";


_formatText("{0}",s);

File: samples/JavaScript/Json Store/Cursor/Select/By Parent.js
session.setAsync(false);

var store = session.getDatabase("playground").getStore("forum");
var s = "";

// Get the first forum document
var doc = store.openCursor().hierarchical(1).findOne().loadDocument();
var unid = doc.getUnid();

s += ">>> Select all documents by parent UNID, "+unid+"\n";
var c = store.openCursor().range(0,5);
c.parentUnid(unid).find(function(e) {
  s += "  value:"+darwino.Utils.toJson(e.getJson(),false)+"\n";
});  
s += "\n";

s += ">>> Select all documents using parent document\n";
var c2 = store.openCursor().range(0,5);
c2.parent(doc).find(function(e) {
  s += "  value:"+darwino.Utils.toJson(e.getJson(),false)+"\n";
});  
s += "\n";

darwino.Utils.setText("content","{0}",s);